### PR TITLE
Khwe 22 render songs for concrete album

### PR DESCRIPTION
**Pull request by OlegShynkarenko**
screencapture-localhost-8080-albums-album4-2018-11-09-20_38_13

### CODE REVIEW

File: src/components/AlbumSongsTable/AlbumSongTable.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import albumSongsTable from "./albumSongsTable.html";
```
Comment (ihor4568) [inline src/components/AlbumSongsTable/AlbumSongTable.js]: please check that folder name is the same as file name and matches class and variable names

File: src/components/Main/Main.js [hunk 2]
```diff
@@ -135,6 +140,11 @@ export class MainComponent {
       return;
     }
 
+    if (/albums\/\w+/.test(pathname)) {
```
Comment (ihor4568) [inline src/components/Main/Main.js]: you could extract song id here and pass it to component in mount() or in other method

Comment (OlegShynkarenko) [reply src/components/Main/Main.js]: Corrected

File: src/components/AlbumSongsTable/AlbumSongTable.js (hunk 1 above)
Comment (OlegShynkarenko) [reply src/components/AlbumSongsTable/AlbumSongTable.js]: Corrected

File: src/components/Main/Main.js [hunk 3]
```diff
@@ -135,6 +141,16 @@ export class MainComponent {
       return;
     }
 
+    if (ALBUMS_URL_REGEXP.test(pathname)) {
```
Comment (ihor4568) [inline src/components/Main/Main.js]: why do you need to extract this as a const?

File: src/components/AlbumSongsTable/AlbumSongsTable.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
+import albumSongsTable from "./AlbumSongsTable.html";
+import { MusicService } from "../../services/MusicService";
+import { SongsTableComponent } from "../SongsTable/SongsTable";
+
+export class AlbumsSongTableComponent {
```
Comment (ihor4568) [inline src/components/AlbumSongsTable/AlbumSongsTable.js]: please rename it to AlbumSongTableComponent (not Albums) to match all other names

File: src/components/AlbumSongsTable/AlbumSongsTable.js [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+      data: this.songs,
+      onSongPlay: this.props.onSongPlay,
+      onSongStop: this.props.onSongStop,
+      playingSongId: this.playingSongId
+    });
+    this.albumSongs.mount();
+  }
+
+  mount(params = {}) {
+    const mountingParameters = {
```
Comment (ihor4568) [inline src/components/AlbumSongsTable/AlbumSongsTable.js]: wat?? you can differentiate from albumId. If albumId was passed - then fetch data, if not - then dont

File: src/components/AlbumSongsTable/AlbumSongsTable.js [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+        });
+        this.mount({ shouldFetchData: false });
+      });
+  }
+
+  fetchAuthorForTitle(albumId) {
+    Promise.all([MusicService.getAlbums(), MusicService.getAuthors()]).then(
+      ([albums, authors]) => {
+        albums.forEach(album => {
+          if (album.id === albumId) {
```
Comment (ihor4568) [inline src/components/AlbumSongsTable/AlbumSongsTable.js]: extract this to separate method

File: src/components/Main/Main.js (hunk 3 above)
Comment (OlegShynkarenko) [reply src/components/Main/Main.js]: Corrected

File: src/components/AlbumSongsTable/AlbumSongsTable.js (hunk 4 above)
Comment (OlegShynkarenko) [reply src/components/AlbumSongsTable/AlbumSongsTable.js]: Corrected

File: package.json [hunk 7]
```diff
@@ -15,11 +15,11 @@
   "license": "ISC",
   "dependencies": {
     "firebase": "5.5.6",
+    "firebase-tools": "6.0.0",
```
Comment (ihor4568) [inline package.json]: why did you move this? firebase-tools is actually a dev dependency

Comment (OlegShynkarenko) [reply package.json]: I didn't move it manually, it might be happend when I reinstall node modules to make prettier works properly.

**Review by ihor4568 — APPROVED** [review]
